client/task-list/tasks/payments/RemotePayments/components/PaymentMethod.js

```diff
@@ -24,7 +26,15 @@ export const PaymentMethod = ( {
 	method,
 	recordConnectStartEvent,
 } ) => {
-	const { key, plugins, title } = method;
+	const {
+		key,
+		plugins,
+		title,
+		post_install_script: postInstallScript,
```

Comment:
This is ok for now, but just noting that this will definitely need to be moved to the payment gateway once https://github.com/woocommerce/woocommerce-admin/pull/6918 and https://github.com/woocommerce/woocommerce-admin/pull/6919 are merged.

Reply:
Yup, agreed. The only downside I see is that it will required too non-parallel requests, instead of one, but it does seem the better place for it.